client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx

```diff
@@ -135,7 +148,8 @@ export const DigiSubDiscountConfirm = () => {
 								</strong>
 								<br />
 								From {nextBillingDate}, your ongoing{' '}
-								{billingPeriod} payment will be {priceDisplay}.
+								{billingPeriod} payment will be {currencySymbol}
+								{formatAmount(discountedPrice)}.
```

Comment:
is this the wrong way round? not sure if the copy here is right, the figma doesn't have much there?